pg/scripts/init.sql

```diff
@@ -52,61 +49,65 @@ create table reactions (
     sabio_rk text
 );
 
+create table organisms (
+    kegg text primary key not null,
+    entry text,
+    name text,
+    taxa text
+);
+
+create table domains (
+    abbreviation text primary key not null,
+    name text
+);
 
+create table genes (
+    kegg text,
+    ncbigene text,
+    uniprot text
+);
 
 -- import data
 copy enzymes from program 'cat /input_data/enzymes/*.txt' delimiter E'\t';
 
 copy compounds from '/input_data/supplementary/compound.txt' delimiter E'\t' CSV HEADER;
 copy ec from '/input_data/supplementary/ec.txt' quote E'\u0007' delimiter E'\t' CSV HEADER;
 copy reactions from '/input_data/supplementary/reaction.txt' delimiter E'\t' CSV HEADER;
-
-
+copy organisms from '/input_data/supplementary/organism.txt' delimiter E'\t' CSV HEADER;
+copy domains from '/input_data/supplementary/domain.txt' delimiter E'\t' CSV HEADER;
+copy genes from program 'cat /input_data/supplementary/gene/*.txt' delimiter E'\t';
 
 -- create lookup indexes
+create index on reactions using gist (name, kegg);
+create index on compounds using gist (name, kegg);
+create index on ec using gist (name, ec);
+create index on organisms using gist (name, kegg);
+create index on domains using gist (name);
+
+-- create indexes for the enzymes table
 create index on enzymes("gene");
 create index on enzymes("organism");
 create index on enzymes("domain");
 create index on enzymes("reaction_id");
 create index on enzymes("compound");
 create index on enzymes using gin (string_to_array(ec_number, ';'));
```

Comment:
Upon reading up on [multicolumn indexes in Postgres](https://www.postgresql.org/docs/current/indexes-multicolumn.html) and research into the performance decrease by @e0 documented by @nanjiangshu, I propose to revert to single-column indexes everywhere. I'm also taking the opportunity to remove the `"` around column names.

```suggestion
create index on reactions using gist (name);
create index on reactions using gist (kegg);
create index on compounds using gist (name);
create index on compounds using gist (kegg);
create index on ec using gist (name);
create index on ec using gist (ec);
create index on organisms using gist (name);
create index on organisms using gist (kegg);
create index on domains using gist (name);

-- create indexes for the enzymes table
create index on enzymes(gene);
create index on enzymes(organism);
create index on enzymes(domain);
create index on enzymes(reaction_id);
create index on enzymes(compound);
create index on enzymes using gin (string_to_array(ec_number, ';'));
```

Reply:
> For the purpose of fuzzy search, I think using a compound column is more appropriate.

I'm not sure I follow. The way I see it, for any of the (reaction, compound, ec, organism), it will rarely be the case  that both _name_ and _kegg_ will be typed in the search. Since people are more likely to search after column decoupled from the other, I expect separate indexes (as they were initially set up I think) to be faster for fuzzy searches.

Reply:
I tried four variations. The one with your suggestion, comma separated (before suggestion), comma separated but primary id first (based on the page you linked to regarding multicolumn gist indexing), and index on compound column (`create index on reactions using gist ((kegg || ' ' || name));`). The performance seems to be about the same for all four actually, they all take around `500ms`. I don't think we have enough data to test this properly. I did some more reading into different implementations and here is a case that's similar to what we had before (comma separated): https://stackoverflow.com/a/29414489

Given the inconclusive results, I can't say any of these are better or worse for our case. I'm fine with merging the suggestion and we can revisit this in the future if needed. Does that sounds good @mihai-sysbio?

Reply:
Many thanks for the super thorough testing @e0. Not sure if you also looked at the CPU usage, if there was any difference.

I'm going to follow the above and commit the suggestion and merge the PR.